### PR TITLE
Add support to Portuguese - Brazil language (pt-BR)

### PR DESCRIPTION
**Pull request by celsofabri**
### Description
- Created new path `/br/` (Portuguese - Brazil);
- Added translations to pt-BR locale;
- Added new Brazilian greetings on the dynamic title.
giphy

### CODE REVIEW

**Comment by celsofabri** [comment]
I loved your site and I would wanna contribute to it. So, I added the Portuguese (Brazil) version! I hope that you like it @notjosh :)

File: locales/br/messages.po [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+
+msgid "example.bad.title"
+msgstr "❌ Não faça isso"
+
+# the placeholder is the year, i.e. "2022"
+msgid "example.bad.body"
+msgstr ""
+"Observe que Keith poderia ter recebido sua resposta minutos antes, e não precisava ter deixado Tim esperando. Na verdade, Tim poderia ter respondido a questão imediatamente!\n"
+"\n"
+"As pessoas que fazem isso geralmente tentam ser educadas, não indo direto ao ponto, como geralmente fariam pessoalmente ou por telefone - e isso é ótimo! Mas é !@#$ e chat não precisa disso. Para a maioria das pessoas, digitar é muito mais lento do que falar. Então, apesar das melhores intenções, **você está apenas fazendo a outra pessoa esperar** para você formular sua pergunta, o que é perda de produtividade (e às vezes um pouco irritante).\n"
```
Comment (angeliski) [inline locales/br/messages.po:41]: Something is wrong here, the year should be in a different format
```suggestion
"As pessoas que fazem isso geralmente tentam ser educadas, não indo direto ao ponto, como geralmente fariam pessoalmente ou por telefone - e isso é ótimo! Mas é %1$s e chat não precisa disso. Para a maioria das pessoas, digitar é muito mais lento do que falar. Então, apesar das melhores intenções, **você está apenas fazendo a outra pessoa esperar** para você formular sua pergunta, o que é perda de produtividade (e às vezes um pouco irritante).\n"
```

Comment (celsofabri) [reply locales/br/messages.po]: Thanks, @angeliski ! For some reason, it changed and I didn't see it.

File: locales/br/messages.po [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+
+msgid "example.bad.message2.timestamp"
+msgstr "14h20m"
+
+# the typo here is intentional - please maintain a similar unprofessional tone!
+msgid "example.bad.message2.body"
+msgstr "que horas será a reunião?"
+
+msgid "example.bad.reply2.timestamp"
+msgstr "14h20m"
```
Comment (vitormattos) [inline locales/br/messages.po:74]:
```suggestion
msgstr "14:20"
```

File: locales/br/messages.po [hunk 3]
```diff
@@ -0,0 +1,133 @@
...
+"- \"Olá! Espero que você esteja bem. Estou atrás do último deck, quando você tiver um tempinho :)\"\n"
+"- \"Hey, se você não estiver ocupado, você poderia atualizar esses cadastros?\"\n"
+"- etc.\n"
+"\n"
+"Isso pode parecer trivial, mas fazer sua pergunta antes de receber a resposta inicial também permite a **comunicação assíncrona**. Se a outra parte estiver ausente e você sair antes que ela volte, ela ainda poderá responder à sua pergunta, em vez de apenas olhar para um \"Olá\" e se perguntar o que perdeu.\n"
+"\n"
+"Se bem feito - todos ficam felizes! 🎉\n"
+
+msgid "example.good.message1.timestamp"
+msgstr "14h15m"
```
Comment (vitormattos) [inline locales/br/messages.po:98]:
```suggestion
msgstr "14:15"
```

File: locales/br/messages.po [hunk 4]
```diff
@@ -0,0 +1,133 @@
...
+msgstr "muito obrigado!"
+
+msgid "example.good.reply2.timestamp"
+msgstr "14h16m"
+
+msgid "example.good.reply2.body"
+msgstr "👌 por nada"
+
+msgid "footer.note"
+msgstr "Isso é meio sério (meio 👀) então por favor não fique [bravo](https://www.youtube.com/watch?v=xzpndHtdl9A) com a pessoa que te enviou aqui."
```
Comment (vitormattos) [inline locales/br/messages.po:123]:
```suggestion
msgstr "Isso é meio sério (meio 👀) então por favor não fique [bravo](https://www.youtube.com/watch?v=xzpndHtdl9A) com a pessoa que te enviou este site."
```

File: locales/br/messages.po [hunk 5]
```diff
@@ -0,0 +1,133 @@
+msgid ""
+msgstr ""
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Language: en\n"
```
Comment (vitormattos) [inline locales/br/messages.po:6]:
```suggestion
"Language: br\n"
```

**Review by vitormattos — CHANGES_REQUESTED** [review]

**Comment by notjosh** [comment]
https://nohello.net/pt-br/ 🥳

**Pull request closed without merging**